nanofi/examples/CMakeLists.txt

```diff
@@ -57,6 +57,8 @@ else ()
     set(LINK_END_FLAGS "")
 endif ()
 
+if (NOT WIN32)
```

Comment:
That means we don't compile nanofi examples for Win at all.
If that's the goal, I would prefer having the Win-specific flags (Wholearchive) removed, too.